pkg/events/domain/certificates.go

```diff
@@ -68,6 +69,10 @@ func (t certificateEventHandler) verify(certificate *x509.Certificate, moment ti
 	return chains[0], nil
 }
 
+func (t certificateEventHandler) VerifiedChain(certificate *x509.Certificate, moment time.Time) ([][]*x509.Certificate, error) {
+	return certificate.Verify(x509.VerifyOptions{Roots: t.trustStore.Pool(), CurrentTime: moment})
```

Comment:
is this right, shouldn't it be split in roots and intermediates?

Reply:
Theoretically yes, but we don't use intermediates anymore... I'll add a note

Reply:
ah yes ofc